SConstruct

```diff
@@ -64,6 +64,15 @@ if not COMMAND_LINE_TARGETS:
     logger.info(__doc__, print_level=False)
     sys.exit(0)
 
+if parse_version(SCons.__version__) < parse_version("3.0.0"):
+    logger.info("Cantera requires SCons with a minimum version of 3.0.0. Exiting.",
+                print_level=False)
+    sys.exit(0)
+
+if os.name not in ["nt", "posix"]:
+    print(f"Error: Unrecognized operating system '{os.name}'")
+    sys.exit(1)
```

Comment:
There isn't, but the check existed before as:
https://github.com/Cantera/cantera/blob/2d211dbca1e3ab6bbf6cf8feeea8d59089c74b1b/SConstruct#L242-L252

Reply:
I think the [only other possibility](https://docs.python.org/dev/library/os.html#os.name) here is `java`, which I guess corresponds to Jython and definitely wouldn't work for compiling Cantera.

Reply:
@speth ... thanks for the comment! My personal preference would be to leave things as is (i.e. not support other OS's), and wait for an enhancement request, rather than potentially receiving an issue where no appropriate exception is raised.